api/src/main/java/org/openmrs/Order.java

```diff
@@ -32,10 +32,18 @@
  * 
  * @version 1.0
  */
-public class Order extends BaseOpenmrsData implements java.io.Serializable {
+public class Order extends BaseOpenmrsData implements java.io.Serializable, Comparable{
 	
 	public static final long serialVersionUID = 4334343L;
-	
+
+	@Override
+	public int compareTo(Object o) {
```

Comment:
OpenmrsUtil has compare method below that will take care of null values for you depending on how you want to treat nulls, so you won't need to check if sequenceNumber is not null and you won't need like 44
`compareWithNullAsGreatest`
`compareWithNullAsLeast`